device/dhcp/clients/v6/dhcpv6client.yaml

```diff
@@ -0,0 +1,140 @@
+components:
+  schemas:
+    Device.Dhcpv6client:
+      description: >-
+        Configuration for emulated DHCPv6 Client on a single Interface. If the DHCPv6 Client receives one or more 
+        DHCPv6 ADVERTISE messages from one or more servers then the client chooses one server from which to request 
+        configuration parameters, based on the configuration parameters offered by the server in the DHCPv6 ADVERTISE 
+        messages. If all configuration parameters match then the first server will be chosen.
+        https://www.rfc-editor.org/rfc/rfc8415.html
+      type: object
+      required: [name]
+      properties:
+        name:
+          x-include: ../../../../common/common.yaml#/components/schemas/Named.Object/properties/name
+          x-field-uid: 1
+        rapid_commit:
+          description: >-
+            If Rapid Commit is set, client initiates Rapid Commit two-message exchange by including Rapid Commit option 
+            in Solicit message.
+          type: boolean
+          default: false
+          x-field-uid: 2
+        ia_type:
+          $ref: '#/components/schemas/Device.Dhcpv6client.IaType'
+          x-field-uid: 3
+        duid_type:
+          $ref: '#/components/schemas/Device.Dhcpv6client.DuidType'
+          x-field-uid: 4
+
+    Device.Dhcpv6client.IaType:
+      type: object
+      properties:
+        choice:
+          description: >-
+            Identity Association: Each IA has an associated IAID. IA_NA and IA_TA options represent different types of 
+            IPv6 addresses and parameters accepted by DHCPv6 clients each used in different context by an IPv6 node. IA_NA 
+            is the Identity Association for Non-temporary Addresses option. IA_TA is the Identity Association for Temporary 
+            Addresses option. IA_PD and IA_NAPD options represent one or more IPv6 prefix and parameters. IA_PD is the Identity 
+            Association for Prefix Delegation and IA_NAPD s the Identity Association for Temporary Prefix Delegation.
+          type: string
+          default: iana
+          x-field-uid: 1
+          x-enum:
+            iana:
+              x-field-uid: 1
+            iata:
```

Comment:
description of `iata` is missing

Reply:
it is just a choice, no timer details for it